src/ButtonComponent/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import cn from 'classnames';
+import React from 'react';
+
+import { ButtonProps } from './types';
+
+const Button = ({ content }: ButtonProps): JSX.Element => {
+  return (
+    <div>
```

Comment:
Why create a `div` element here, when you can just return the `button` itself?

Reply:
it was for test something, I'm going to remove it